### PR TITLE
CHANGELOG.md: update to a new format.

### PR DESCRIPTION
**Pull request by ipochi**
Something similar to 
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.18.md

Signed-off-by: Imran Pochi <imran@kinvolk.io>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,39 @@
...
+      - [AKS](#aks)
+    - [Security](#security)
+    - [Bugfixes](#bugfixes)
+    - [Development](#development)
+    - [Miscellaneous](#miscellaneous)
+- [v0.2.0 - 2020-06-19](#v020---2020-06-19)
+  - [Changes in v0.2.0](#changes-in-v020)
+    - [Kubernetes updates](#kubernetes-updates)
+    - [Platform updates](#platform-updates)
+      - [AKS](#aks)
```
Comment (invidian) [inline CHANGELOG.md:13]: This anchor is wrong. It points to AKS section in v0.2.1

**Comment by invidian** [comment]
@ipochi what should we do with this PR? Are you going to update it?

**Comment by ipochi** [comment]
> @ipochi what should we do with this PR? Are you going to update it?

@invidian I'll look into the autmated creation of the TOC. For now its on the back burner.

I'll close this and create one later, once I am actively on it.

**Comment by invidian** [comment]
Shall we keep the branch then?

**Pull request closed without merging**